### ISSUE TITLE
Unblock UI thread by shifting initialisation to another thread 

### ISSUE THREAD

**Issue by viveksb007** [fossasia/pslab-android#205]
**Actual Behaviour**

Currently initialisation takes place on UI thread that blocks/freezes the UI.

**Expected Behaviour**

Initialisation should take place from another thread which should run in background and update status on UI thread.

**Steps to reproduce it**

Open App, you will see that UI freezes for 3-4 seconds if PSLab Device is connected.

**LogCat for the issue**

None

**Would you like to work on the issue?**

Yes, threads and synchronisation is a bit complex to it might take some time.

**Comment by viveksb007** [comment]
@jithinbp @akarshan96 @asitava1998 here is link to [apk](https://drive.google.com/open?id=0B30IF3fyX4Q3b25kTmxFTnBGTG8) . I have moved initialisation to another thread. Now UI won't be freezed during initialisation.
Could you guys test and revert back in case of any crash or some improvement is needed?
For now User is indicated by textView like ```Initialising Wait..```. Would open another issue for initialising indication to user using progress bar.

**Comment by viveksb007** [comment]
Fixed in #210
